src/repository.py

```diff
@@ -63,6 +67,7 @@ def get_all_loan_ammount_sum():
 		except Exception as e:
 			raise Exception("Error: ", e)
 
+
 	@staticmethod
 	def add_loan_deposit(deposit_amt, loan_id):
 		try:
```

Comment:
commented code at line 76 can be ignored

![image](https://user-images.githubusercontent.com/70574489/217151258-279ed738-ac09-4295-bb69-f17850d91e3d.png)